public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/helpers.tsx

```diff
@@ -36,6 +37,24 @@ export function formikToDetectorName(title) {
   return detectorName;
 }
 
+export async function isExceededMaxAssociatedCount(visId, savedObjLoader) {
```

Comment:
I don't see this fn being used?

Reply:
forgot to clean up this part of code.. removed